devel/n/Portfile

```
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        tj n 9.1.0 v
+github.setup        tj n 9.2.0 v
```

Please add a `revision` field as well:

```suggestion
github.setup        tj n 9.2.0 v
revision            0
```